src/UKHO.S100PermitService.Common/Utilities/PermitReaderWriter.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Text;
+using System.Xml.Serialization;
+using System.Xml;
+using System.IO.Abstractions;
+using UKHO.S100PermitService.Common.Models;
+using System.Diagnostics.CodeAnalysis;
+
+namespace UKHO.S100PermitService.Common.Utilities
```

Comment:
Please avoid namespaces like 'Utilities' and 'Helpers'. It should have a clear intent in terms of the domain model, so in this case something like 'IO' would be appropriate?

Reply:
Renamed the namespaces